pyproject.toml

```diff
@@ -30,6 +30,9 @@ greenlet = "^0.4.15"
 tenacity = "^5.0"
 docopt = "^0.6.2"
 pyyaml = "^5.1"
+sphinx = "^1.8"
```

Comment:
I think these should be added as `dev` dependencies, i.e. a regular user installing Zelt wouldn't need to build its documentation.